src/get-glamor-classname.js

```diff
@@ -1,4 +1,4 @@
-import {css, styleSheet} from 'glamor'
+import {css} from 'glamor'
```

Comment:
I _love_ not having to reach into the glamor to get the `styleSheet`. This is awesome! :clap: